api/src/pcapi/core/mails/transactional/users/ubble/ubble_ko_reminder.py

```diff
@@ -3,17 +3,56 @@
 
 from dateutil.relativedelta import relativedelta
 
+from pcapi.core import mails
 import pcapi.core.fraud.models as fraud_models
 import pcapi.core.fraud.ubble.api as ubble_api
+import pcapi.core.mails.models as mails_models
 import pcapi.core.subscription.api as subscription_api
 import pcapi.core.subscription.models as subscription_models
 import pcapi.core.subscription.ubble.api as ubble_subscription
 import pcapi.core.users.models as users_models
 
+from . import constants
+
 
 logger = logging.getLogger(__name__)
 
 
+def send_ubble_ko_reminder_emails() -> None:
+    users = find_users_that_failed_ubble_check_seven_days_ago()
+    users_by_reason_codes = sort_users_by_reason_codes(users)
+
+    result = {}
+
+    for reason_code, users in users_by_reason_codes.items():
+        data = get_ubble_ko_reminder_email_data(reason_code)
+        if not data:
+            if reason_code == "no_reason_code":
+                logger.warning(
+                    "Could not find reason code for users that failed ubble check",
+                    extra={"users": [user.id for user in users]},
+                )
+            else:
+                logger.warning("Could not find email template for reason code %s", reason_code)
+            continue
+
+        result[reason_code] = mails.send(recipients=[user.email for user in users], data=data)
+
+    if all(result.values()):
+        logger.info("Sent all ubble ko reminder emails successfully", extra={"result": result})
+    else:
+        logger.error("Could not send some ubble ko reminder emails", extra={"result": result})
```

Comment:
est-ce qu'on va faire quelque chose de cette erreur ?

Reply:
Yes
Si ça apparait il y a une erreur qu'on ne gère pas. Il faut l'ajouter ou corriger

Reply:
la raison que tu évoques correspond plutôt au log plus hat `logger.warning("Could not find email template for reason code %s", reason_code)`, non ?
là c'est des erreurs d'envoi de mail